lua/graph.lua

```diff
@@ -823,9 +843,6 @@ function filter_tags_generic(kv)
 
   if kv["bike_backward"] == "true" then
     oneway_bike = oneway[kv["oneway:bicycle"]]
-    if (oneway_bike == "false" and kv["bicycle:backward"] == "yes") then
```

Comment:
The only way that this if condition would be fulfilled is if a way had both the "oneway:bicycle=no" tag and the "bicycle:backward=yes" which it seems there are no roads where this is true so I cannot provide before and after results. But even if there were a road like this, why would we set it to be a oneway for bikes? both tags specify that it is not a oneway for bikes so it doesn't make sense to me that we would set oneway_bike = true (which later on would make bike_forward = false)